doc/developer_guide/psyir.rst

```diff
@@ -273,6 +273,22 @@ See the full Container API in the
 `Container reference guide <https://psyclone-ref.readthedocs.io/en/latest/
 autogenerated/psyclone.psyir.nodes.html#psyclone.psyir.nodes.Container>`_.
 
+FileContainer
+^^^^^^^^^^^^^
+
+The `FileContainer` node is a subclass of the `Container` node and is
+used to capture the concept of a file that contains one or more
+`Container` and/or `Routine` nodes. Whilst this structure is the same
+as for a `Container`, it is useful to distinguish between the two as
+backends may need to deal differently with a `FileContainer` and a
+`Container`. A file container is always created at the root of the
+PSyIR tree when parsing Fortran code, as Fortran code has the concept
+of a program (captured as a `FileContainer`) that can contain one or
```

Comment:
I know what you mean but this can be confusing as the program keyword is captured as a Routine with is_program attribute.

Maybe simplify the sentence to something like:
"A `FileContainer` is always created at the root of the PSyIR tree when parsing Fortran code, as Fortran can contain one or more program units (captured as `Containers` and/or `Routines`)."

Reply:
Simpler thanks.